features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -1535,13 +1535,15 @@ IDN_CONFIG_RESOURCE (ID) ON DELETE CASCADE ON UPDATE RESTRICT
 CREATE TABLE IDN_REMOTE_FETCH_CONFIG(
 	ID VARCHAR(255) NOT NULL,
     TENANT_ID INTEGER NOT NULL,
-    IS_ENABLED CHAR(1) NOT NULL,
+	IS_ENABLED CHAR(1) NOT NULL,
```

Comment:
formatting issue

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/3143